exampleSite/content/posts/theme-documentation-basics.md

```diff
@@ -213,6 +213,14 @@ If you want to display posts on the homepage, the options are:
   showPostsOnHomePage = "popular"
 ```
 
+### Date format 
+Change date format 
```

Comment:
Change to:
The date format being used across can be chaged. For valid date strings see: https://pkg.go.dev/time#pkg-constants